helm/helm-release.bzl

```diff
@@ -102,6 +104,8 @@ helm_release = rule(
       "sops_yaml": attr.label(allow_single_file = True, mandatory = False),
       "helm_version": attr.string(mandatory = False),
       "kubernetes_context": attr.string(mandatory = False),
+      "create_namespace": attr.string(mandatory = False, default = ""),
+      "wait": attr.string(mandatory = False, default = ""),
```

Comment:
These changes are not reflected in the README, maybe the documentation should be moved to [stardoc](https://github.com/bazelbuild/stardoc)?